lib/networkCall.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios')
+const logger = require('../utils/logger')
+
+const networkCall = async (url, queryParams, method = 'get', body = null, headers = null) => {
```

Comment:
Please add default value for `queryParams` too

Reply:
done.